rust+wasm/{{project-name}}-wasm/README.md

```diff
@@ -0,0 +1,152 @@
+      <img src="https://img.shields.io/badge/License-MIT-blue.svg" alt="License-MIT">
+    </a>{% endif %}
+    <a href="https://docs.rs/{{project-name}}">
+      <img src="https://img.shields.io/static/v1?label=Docs&message=docs.rs&color=blue" alt="Docs">
+    </a>{% if discord != "" %}
+    <a href="{{discord}}">
+      <img src="https://img.shields.io/static/v1?label=Discord&message=join%20us!&color=mediumslateblue" alt="Discord">
+    </a>{% endif %}
+  </p>
+</div>
+
+<div align="center"><sub>:warning: Work in progress :warning:</sub></div>
+
+##
+
+Description.
+
+## Outline
+
+- [Set-up](#set-up)
+- [Build for Javascript](#build-for-javascript)
+- [Testing the Project](#testing-the-project)
+- [Publishing a Package](#publishing-a-package)
+- [License](#license)
+
+## Set-up
+{% if nix == false %}
+- Install [`wasm-pack`][wasm-pack]
+
+  ```console
+  cargo install wasm-pack
+  ```
+{% endif %}
+We'll use [`wasm-pack`][wasm-pack] for building, testing, and publishing
+our wasm project.
+
+### Build for Javascript
+
+The `wasm-pack build` command will compile the code in this directory into
+wasm and generate a `pkg` folder by default, containing the wasm binary, a
+Javascript-wrapper file, the {{project-name}}-wasm README (and version), and a
+`package.json` file.
+
+- Targetting node:
+
+  ```console
+  wasm-pack build --target nodejs
+  ```
+
+- Targetting browswers:
+
+  ```console
+  wasm-pack build --target web
+  ```
+
+- Targetting bundlers like [webpack][webpack]:
+
+  ```console
+  wasm-pack build --target bundler
+  ```
+
+## Testing the Project
+
+For running tests in the current directly, use this command:
+
+```console
+wasm-pack test
+```
+
+- Run tests expected to execute in [Node.js][node-js]:
+
+```console
+wasm-pack test --node
+```
+
+- Run browser tests in a headless browwer:
+
+```console
+wasm-pack test --headless --firefox --chrome --safari
+```
+
+*Note*: Make sure you have the appropriate browser installed when running
+locally.
+
+## Publishing a Package
+
+Once you've [built the package](#build-for-javascript), which lives under
+`pkg` by default (or a sub-directory of your choosing), you can pack and
+publish it via (given credentials):
+
+```console
+wasm-pack publish
+```
+
+## License
+{% if license == "Apache" %}
+This project is licensed under the [Apache License 2.0](./LICENSE), or
+[http://www.apache.org/licenses/LICENSE-2.0][apache].
+{% elsif license == "MIT" %}
+This project is licensed under the [MIT License](./LICENSE),
+or [http://opensource.org/licenses/MIT][mit].
+{% else %}
+This project is licensed under either of
+
+- Apache License, Version 2.0, ([LICENSE-APACHE](./LICENSE-APACHE) or http://www.apache.org/licenses/LICENSE-2.0)
+- MIT license ([LICENSE-MIT](./LICENSE-MIT) or http://opensource.org/licenses/MIT)
+
+at your option.
+
+### Contribution
+
+Unless you explicitly state otherwise, any contribution intentionally
+submitted for inclusion in the work by you, as defined in the Apache-2.0
```

Comment:
Missed the conditional here

Reply:
This is appropo for the dual license it seems: https://github.com/rustwasm/wasm-pack-template#license.